rest_framework_json_api/parsers.py

```diff
@@ -4,6 +4,8 @@
 from rest_framework import parsers
 from rest_framework.exceptions import ParseError
 
+from django.conf import settings
```

Comment:
Could you move this import line above the `rest_framework` lines without a newline between them? This project groups third party libraries together alphabetically.